tools/c7n_azure/c7n_azure/resources/sqldatabase.py

```diff
@@ -23,8 +23,8 @@ class resource_type(ArmResourceManager.resource_type):
         service = 'azure.mgmt.sql'
         client = 'SqlManagementClient'
         enum_spec = ('databases', 'list_by_server', {
-            'resource_group_name': 'resourceGroup',
-            'server_name': 'name'
+            'resource_group_name': lambda p: p['resourceGroup'],
```

Comment:
per previous comment, strong preference for class/staticmethod on metadata for this purpose instead of lambda funcs in datastructs.